docs/intree-migrate.md

```diff
@@ -0,0 +1,229 @@
+  value of `adminId` in the in-tree Storageclass is different from value `admin`
+  , the `adminSecretName` mentioned in the in-tree Storageclass has to be
+  patched with the base64 value of the `adminId` parameter value, otherwise this
+  step can be skipped.
+
+In below examples, `fast-rbd` is in-tree storageclass with provisioner
+referencing in-tree provisioner `kubernetes.io/rbd`.
+
+```console
+$ kubectl describe sc fast-rbd |grep -i provisioner
+Provisioner:           kubernetes.io/rbd
+```
+
+## Existing in-tree volume operations
+
+### Mount volume to a POD
+
+- List existing in-tree PVC and PV
+
+```console
+$ kubectl get pvc,pv
+NAME                             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/existpvc   Bound    pvc-c827c204-94e7-4ed9-87e9-fce212fe61a0   4Gi        RWO            fast-rbd       7m17s
+
+NAME                                                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS   REASON   AGE
+persistentvolume/pvc-c827c204-94e7-4ed9-87e9-fce212fe61a0   4Gi        RWO            Delete           Bound    default/existpvc   fast-rbd                7m29s
+
+```
+
+- Create a pod with the PVC and verify mount
+
+```
+$ kubectl create -f pod-in.yaml
+pod/in-task-pv-pod created
+
+$ kubectl get pods
+NAME            READY STATUS  RESTARTS AGE
+in-task-pv-pod  1/1   Running 0       6m16
+
+$ mount |grep rbd
+/dev/rbd0 on /var/lib/kubelet/plugins/kubernetes.io/csi/pv/pvc-c827c204-94e7-4ed9-87e9-fce212fe61a0/globalmount/mig_mons-b7f67366bb43f32e07d8a261a7840da9_image-31a456d4-3a76-4123-8ced-1d8d6aa64f17_7265706c696361706f6f6c type ext4 (rw,relatime,stripe=16,_netdev)
+/dev/rbd0 on /var/lib/kubelet/pods/058fbe5c-02ed-4c95-9707-b992599c6728/volumes/kubernetes.io~csi/pvc-c827c204-94e7-4ed9-87e9-fce212fe61a0/mount type ext4 (rw,relatime,stripe=16,_netdev)
+
+```
+
+### Unmount volume
+
+- Delete Pod and Verify unmount has happened successfully
+
+```console
+$ kubectl delete pod in-task-pv-pod
+pod "in-task-pv-pod" deleted
+
+$ mount |grep rbd
+$
+```
+
+### Delete Volume
+
+- Delete existing in-tree PVC
+
+```console
+$ kubectl get pvc,pv
+NAME                             STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+persistentvolumeclaim/existpvc   Bound    pvc-c827c204-94e7-4ed9-87e9-fce212fe61a0   4Gi        RWO            fast-rbd       7m50s
+
+NAME                                                        CAPACITY   ACCESS MODES   RECLAIM POLICY   STATUS   CLAIM              STORAGECLASS   REASON   AGE
+persistentvolume/pvc-c827c204-94e7-4ed9-87e9-fce212fe61a0   4Gi        RWO            Delete           Bound    default/existpvc   fast-rbd                8m2s
+
+$ kubectl delete pvc existpvc
+persistentvolumeclaim "existpvc" deleted
+
+$ kubectl get pvc,pv
+No resources found
+$
+```
+
+### New Volume Operations
+
```

Comment:
explain that these operations are not possible with the in-tree driver? although I am wondering what new is with "create volume".

Reply:
its covered in the design doc which is referred from there. this usage doc list the operations listed in the design doc.

Reply:
also clarfied it in the better words in last refresh. ptal.. thanks.